tests/results.txt

```diff
@@ -2237,8 +2237,9 @@ T{ saved-string s\"  ok\ned: \nq  ok\nrestore-output  " compare -> 0 }T  ok
 \ Test --- q --- Don't quit if we have unsaved changes  ok
 test-ed  ok
 saved-string dump 
-0F8E  0A 61 20 0A 7A 7A 7A 20  0A 2E 20 0A 71 20 0A 3F  .a .zzz  .. .q .?
-0F9E  0A 51 20  .Q  ok
+0EC0  0A 61 20 0A 7A 7A 7A 20  0A 2E 20 0A 71 20 0A 3F  .a .zzz  .. .q .?
+0ED0  0A 51 20                                          .Q 
```

Comment:
i added the space alignment (purely cosmetic) and trailing newline for consistency

Reply:
I like it.  That's one of those things that never bothered me quite enough to fix it, but it makes it look much nicer.